### ISSUE TITLE
Install buildkite repository when installing deb/rpm file

### ISSUE THREAD

**Issue by BernatBC** [sourcegit-scm/sourcegit#820]
Right now, if someone wants to use the buildkite repo for automatic updates, he needs to add it using the instructions in the readme. However, it's possible to install the repository when installing the deb/rpm file. 

Normally, in deb files, a `postinst` script is added with the commands of the repository installation. While in rpm files, a `%post` section is added with the commands.

**Comment by BernatBC** [comment]
If I have time, I could try it to add it myself

**Comment by aikawayataro** [comment]
I do not think this is a reasonable thing to do. Why would you need to install repositories on package installation if you install repositories first to install the package?

> Normally, in deb files, a postinst script is added with the commands of the repository installation. While in rpm files, a %post section is added with the commands.

I have never seen this

**Comment by gadfly3173** [comment]
> I have never seen this

In fact, both vscode and chrome deb do this.

![image](https://github.com/user-attachments/assets/0a8768fb-cb32-4c57-adfc-7e6eaf386bec)

**Comment by aikawayataro** [comment]
> vscode

I see, it will ask to install repository after installation. The prompt to the user during the installation looks good for me, implicit installation does not.
